pkg/functionconfig/types.go

```diff
@@ -99,6 +100,8 @@ const (
 
 	// ExplicitAckModeExplicitOnly allows only explicit acks and disables implicit acks
 	ExplicitAckModeExplicitOnly ExplicitAckMode = "explicitOnly"
+
+	DefaultWorkerTerminationTimeout string = "30s"
```

Comment:
add explanation, e.g.:

// DefaultWorkerTerminationTimeout bla bla

Reply:
Id change default to be 0 (for BC) and set it to 30 on kafka + explicit ack on kafka trigger